frontend/src/features/admin-form/responses/InsightsPage/UnlockedInsights/TableChart.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@chakra-ui/react'
+
+export const TableChart = ({ data }: { data: [string, number | string][] }) => {
+  return (
+    <TableContainer>
+      <Table variant="simple" my="1rem">
+        <Thead>
+          <Tr>
+            <Th>Answer</Th>
+            <Th isNumeric>Count</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {data.map((val, idx) => {
+            if (typeof val[1] === 'number')
+              return (
+                <TableChartRows
+                  answer={val[0]}
+                  value={Number(val[1])}
+                  key={idx}
```

Comment:
can use "answers" as keys to avoid potential issues with component state and DOM updates?